source/2019-08-16-the-ember-times-issue-111.md

```diff
@@ -97,13 +98,13 @@ and last, but not least,
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [Ember Table 2.0 Release 🎉](https://medium.com/build-addepar/ember-table-2-0-release-5b91903f3e34)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+[Ember Table 2.0](https://github.com/Addepar/ember-table) (“ET2”) has been rewritten from the ground up to be even more performant and flexible to use cases and was officially launched! [Ember Table 1.0](https://github.com/Addepar/ember-table-legacy) was then renamed “ember-table-legacy” to avoid confusion.
```

Comment:
```suggestion
[Ember Table 2.0](https://github.com/Addepar/ember-table) (“ET2”) has been rewritten from the ground up to be even more performant and flexible to use cases and was officially launched! [Ember Table 1.0](https://github.com/Addepar/ember-table-legacy) has been renamed to “ember-table-legacy” to avoid confusion.
```